smarts/sstudio/types.py

```diff
@@ -813,6 +813,36 @@ def to_geometry(self, road_map: Optional[RoadMap] = None) -> Polygon:
             poly = shapely_rotate(poly, self.rotation, use_radians=True)
         return shapely_translate(poly, xoff=x, yoff=y)
 
+@dataclass(frozen=True)
+class ConfigurableZone(Zone):
+    """A descripter that defines a specific configurableZone defined by user"""
+
+    ext_coordinates: List[Tuple[float, float]]
```

Comment:
Please add a `__post_init__` that can check that assert that there are a minimum of 2 points provided.